commands/serve.js

```diff
@@ -1,4 +1,23 @@
 'use strict'
 
+const serve = require('../lib/serve.js')
+
 module.exports = function (input, flags, logger, options) {
+  const cwd = options.cwd
+  return serve.startServer({
+    cwd,
+    port: 3000
```

Comment:
There'll probably be a `--port` flag or something later. I've also contemplated automatically finding and binding to a free port.